src/Microsoft.DotNet.Interactive.Tests/LanguageKernelPackageTests.cs

```diff
@@ -116,41 +116,94 @@ public async Task it_can_load_assembly_references_using_r_directive_separate_sub
         {
-            var workingDirectory = Directory.GetCurrentDirectory();
-            DisposeAfterTest(() => Directory.SetCurrentDirectory(workingDirectory));
+            var currentDirectory = Directory.GetCurrentDirectory();
+            DisposeAfterTest(() => Directory.SetCurrentDirectory(currentDirectory));
 
             var kernel = CreateKernel(language);
 
-            if (changeWorkingDirectory)
+            if (changeCurrentDirectoryInUserCode)
             {
+                //Even when a user changes the current directory, loading from a relative path is not affected.
                 await kernel.SendAsync(new SubmitCode("System.IO.Directory.SetCurrentDirectory(\"..\")"));
             }
 
-            var fullName = new FileInfo(typeof(JsonConvert).Assembly.Location).FullName;
+            var dllName = new FileInfo(typeof(JsonConvert).Assembly.Location).Name;             
 
-            var currentDirectoryName = new DirectoryInfo(Directory.GetCurrentDirectory()).Name;
+            var code = language switch
+            {
+                Language.CSharp => $"#r \"{dllName}\"",
+                Language.FSharp => $"#r \"{dllName}\""
+            };
 
-            var relativeDllPath = Path.GetRelativePath(
-                Directory.GetCurrentDirectory(),
-                fullName);
+            var command = new SubmitCode(code);
 
-            var relativePath =
-                Path.Combine(
-                    "..",
-                    currentDirectoryName,
-                    relativeDllPath)
-                .Replace("\\", "/");
+            await kernel.SendAsync(command);
 
-            var code = language switch
+            KernelEvents.Should()
+                        .ContainSingle<CommandSucceeded>(c => c.Command == command);
+        }
+
+        [Theory]
+        [InlineData(Language.CSharp, false)]
+        [InlineData(Language.FSharp, false)]
+        [InlineData(Language.CSharp, true)]
+        [InlineData(Language.FSharp, true)]
+        public async Task it_can_load_script_files_using_load_directive_with_relative_path(Language language, bool changeCurrentDirectoryInUserCode)
+        {            
+            var currentDirectory = Directory.GetCurrentDirectory();
+            DisposeAfterTest(() => Directory.SetCurrentDirectory(currentDirectory));
+
+            var kernel = CreateKernel(language);
+
+            if (changeCurrentDirectoryInUserCode)
             {
-                Language.CSharp => $"#r \"{relativePath}\"",
-                Language.FSharp => $"#r \"{relativePath}\""
+                //Even when a user changes the current directory, loading from a relative path is not affected.
+                await kernel.SendAsync(new SubmitCode("System.IO.Directory.SetCurrentDirectory(\"..\")"));
+            }
+
+            await kernel.SendAsync(new SubmitCode("Environment.CurrentDirectory"));
```

Comment:
This doesn't look necessary for the test.

Reply:
you are right, probably a leftover from debugging...
fixed in https://github.com/dotnet/interactive/pull/1176/commits/0dd55f30e66ea43e619e2fa75ba4cccdbbae6f6e